docs/src/main/asciidoc/datasource.adoc

```diff
@@ -436,69 +421,77 @@ AgroalDataSource usersDataSource;
+Integration is automatic if the Narayana JTA extension is also available.
 
-=== Named Datasources
+You can override this by setting the `transactions` configuration property.
+See the <<configuration-reference, Configuration Reference>> section below.
 
-When using Dev Services the default datasource will always be created, but to specify a named datasource you need to have
-at least one build time property so Quarkus knows to create the datasource. You will usually either specify
-the `db-kind` property, or explicitly enable Dev Services: `quarkus.datasource."name".devservices.enabled=true`.
+==== Named datasources
 
-[[in-memory-databases]]
-== Testing with in-memory databases
+When using Dev Services, the default datasource will always be created, but to specify a named datasource, you need to have at least one build time property so Quarkus can detect how to create the datasource.
 
-Some databases like H2 and Derby are commonly used in "embedded mode" as a facility to run quick integration tests.
+You will usually specify the `db-kind` property or explicitly enable Dev Services by setting `quarkus.datasource."name".devservices.enabled=true`.
 
-Our suggestion is to use the real database you intend to use in production; container technologies made this simple enough so you no longer have an excuse. Still, there are sometimes
-good reasons to also want the ability to run quick integration tests using the JVM powered databases,
-so this is possible as well.
+[[in-memory-databases]]
+=== Testing with in-memory databases
+
+Some databases like H2 and Derby are commonly used in the _embedded mode_ as a facility to run integration tests quickly.
 
-It is important to remember that when configuring Derby to use the embedded engine,
-this will work as usual in JVM mode but such an application will not compile into a native executable, as the Quarkus extensions only cover for making the JDBC client code compatible with the native compilation step: embedding the whole database engine into a native executable is currently not implemented.
+It is recommended to use the real database you intend to use in production, but it’s possible to use JVM-powered databases for the scenarios when the ability to run quick integration tests is required.
 
-Compiling H2 into a native-image is now possible (it didn't use to be) but this is considered experimental; even if this is technically possible,
-we would still recommend to consider carefully why you would want to embed H2 within your native-image.
+Note that while configuring Derby to use the embedded engine works as usual in JVM mode, such an application will not compile into a native executable, because the Quarkus extensions only cover making the JDBC client code compatible with the native compilation step.
+Embedding the entire database engine into a native executable is currently not implemented.
```

Comment:
This probably shouldn't be two sentences, as the explaination from the first sentence is in the second one.

Maybe:

```suggestion
Note that while configuring Derby to use the embedded engine works as usual in JVM mode, such an application will not compile into a native executable, because the Quarkus Derby extensions does not support embedding the entire database engine into a native executable: it only cover making the JDBC client code compatible with the native compilation step.
```

Reply:
Thank you! Will apply this in slightly modified version